Makefile

```diff
@@ -342,7 +342,10 @@ deploy-autoscaler-bosh:
 deploy-cleanup:
 	${CI_DIR}/autoscaler/scripts/cleanup-autoscaler.sh;
 
-
+PHONY: download-java-bosh-package
+download-java-bosh-package:
+	echo " - Download JDK dependency autoscaler"
+	DEBUG="${DEBUG}" ${CI_DIR}/autoscaler/tasks/update-sdk/download_java.sh
```

Comment:
If this produces a file as a result of a download, I propose to add a non-PHONY make-target for that file that then takes the recipe. `download-java-bosh-package` then would just stay without recipe and only depend on that file.

Reply:
Nice catch! This target is not needed anymore. I forgot to remove that.